.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -9,30 +9,52 @@ DELETE ME
 
 Use the search function to see if someone else has already submitted the same bug report.
 
+Try to describe the problem with as much detail as possible.
+
+Some bugs may only occur on certain devices or versions of Android. Please add information about your device and the version of Android that is running on it (you can look these up under `Settings → About Phone`), as well as which version of AntennaPod you are using.
+
+If the bug only seems to occur with a certain podcast, please include the URL of that podcast.
+
+If possible, add instructions on how to reproduce the bug.
+
+If possible, add a logfile to your post. This is especially useful if the bug makes the application crash. You can create logfiles with apps like `aLogcat`. Just launch `alogcat`, then start AntennaPod and reproduce the bug. aLogcat should now display information about the bug when you start it. AntennaPod also has an `export logs` feature.
```

Comment:
Please remove aLogcat. On newer Android versions, it only works with root access while AntennaPod's own log feature works (for AntennaPod's own logs). The part about reproducing the bug sounds good, so that can be kept :+1: